### PR TITLE
More README updates

### PR DESCRIPTION
**Pull request by kohr-h**
I have made some more edits, in particular the "Getting Involved" section that I've tried to flesh out a bit. Check if you like it -- since you have a better overview of how things work, you may want to change things accordingly.

### CODE REVIEW

File: contrib/clojure-package/README.md [hunk 1]
```diff
@@ -6,23 +6,38 @@ A Clojure Package Built on the MXNet Deep Learning Library
...
+
+That said, because it closely tracks the Scala package, Clojure MXNet can be expected to have a similar level of maturity and stability regarding the low-level functionality. It is mostly in the hand-written Java interop part of the Clojure wrapper where bugs are more likely to be encountered. Such bugs tend to be fixed rather quickly once they are known and their origin is clear (see also [Getting Involved](#getting-involved)).
+
+For an overview of the development status and open problems, please refer to [Clojure Package Contribution Needs](https://cwiki.apache.org/confluence/display/MXNET/Clojure+Package+Contribution+Needs).
+
+## Getting Involved
+
+By far the best way to get involved with this project is to install the Clojure MXNet package, run the examples, play around, build new things with it, and get back to the development team with feedback! Your input can not only help to identify current issues, but also guide the future development of the Clojure package by pointing out must-have features that are currently missing, or by identifying usability or performace problems of high impact.
```
Comment (kohr-h) [inline contrib/clojure-package/README.md:25]: I wasn't sure if I should use "we" or "us".

Comment (gigasquid) [reply contrib/clojure-package/README.md:25]: I think it sounds fine as is - but us also works

File: contrib/clojure-package/README.md [hunk 2]
```diff
@@ -6,23 +6,38 @@ A Clojure Package Built on the MXNet Deep Learning Library
...
+
+There are two main ways of reaching out to other users and the package maintainers:
+
+- If you have a question or general feedback, or you encountered a problem but are not sure if it's a bug or a misunderstanding, then the *Apache Slack* (channels `#mxnet` and `#mxnet-scala`) is the best place to turn check out. To join, [ask for an invitation](https://mxnet.apache.org/community/contribute.html#slack) at `dev@mxnet.apache.org`.
+- If you found a bug, miss an important feature or want to give feedback directly relevant for development, please head over to the MXNet [GitHub issue page](https://github.com/apache/incubator-mxnet/issues) and create a new issue. If the issue is specific to the Clojure package, consider using a title starting with `[Clojure]` to make it easily discoverable among the many other, mostly generic issues.
```
Comment (gigasquid) [inline contrib/clojure-package/README.md:30]: Great point about the Clojure title 💯

File: contrib/clojure-package/README.md (hunk 1 above)
Comment (gigasquid) [inline contrib/clojure-package/README.md:25]: This section is great!

File: contrib/clojure-package/README.md [hunk 3]
```diff
@@ -6,23 +6,38 @@ A Clojure Package Built on the MXNet Deep Learning Library
...
+For a more comprehensive overview of different ways to contribute, see [Contributing to MXNet](https://mxnet.apache.org/community/contribute.html).
 
 ## Getting Started
 
 The Clojure MXNet framework consists of a core C library, a Scala API that talks to the core through [JNI (Java Native Interface)](https://en.wikipedia.org/wiki/Java_Native_Interface) bindings, and finally a Clojure wrapper around the Scala API.
 
-Since there is native code involved in the framework, what OS you are running matters.
-
-The following combinations of operating system and compute device are supported:
+Since the core contains native (compiled) code and is bundled with the language bindings, your hardware and OS matter to the choices to be made during installation. The following combinations of operating system and compute device are supported:
```
Comment (gigasquid) [inline contrib/clojure-package/README.md:40]: Nice wording

**Review by gigasquid — APPROVED** [review]

**Comment by gigasquid** [comment]
Looks great. Thanks tons for the improvements

**Comment by kohr-h** [comment]
Cool, thanks for the quick reply!

**Pull request merged**